omnibus/config/software/more-ruby-cleanup.rb

```diff
@@ -48,7 +47,6 @@
       ed25519.png
       example
       examples
-      ext
```

Comment:
Was this for nokogiri or the http gem mess. This is going to bloat the builds pretty significantly.

Reply:
This was for the http gem - specifically without this it is missing `http-parser-ext`.